docusaurus.config.ts

```diff
@@ -76,6 +76,14 @@ const config: Config = {
       } satisfies Preset.Options,
     ],
   ],
+  plugins: [
+    [
+      '@docusaurus/plugin-google-tag-manager',
+      {
+        containerId: 'GTM-5X4J3CF7',
```

Comment:
can be put in env vars (I don't have Github permissions) but not a big deal cuz eventually it's embedded inside the HTML anyways and sent over the public network as is, so can be found easily if one is interested
The only risk here - is if someone copies this code and mistakenly spams us with wrong data..